AIX build fails to run with CGO_ENABLED=0

**Describe the bug**
See https://github.com/open-telemetry/opentelemetry-collector/actions/runs/13653662476/job/38167780904?pr=12540

On update to v4.25.2, we see a compilation error for aix:
```
pushd cmd/otelcorecol && CGO_ENABLED=0 go build -trimpath -o ../../bin/otelcorecol_aix_ppc64 \
	-tags  ./cmd/otelcorecol && popd
~/work/opentelemetry-collector/opentelemetry-collector/cmd/otelcorecol ~/work/opentelemetry-collector/opentelemetry-collector
# github.com/shirou/gopsutil/v4/net
Error: /home/runner/go/pkg/mod/github.com/shirou/gopsutil/v4@v4.25.2/net/net_aix_nocgo.go:[9](https://github.com/open-telemetry/opentelemetry-collector/actions/runs/13653662476/job/38167780904?pr=12540#step:5:10)2:10: not enough return values
	have ([]IOCountersStat)
	want ([]IOCountersStat, error)
make: *** [Makefile:159: otelcorecol] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -34,6 +34,7 @@ build_test:  ## test only buildable
 	GOOS=netbsd go test ./... | $(BUILD_FAIL_PATTERN)
 	GOOS=openbsd go test ./... | $(BUILD_FAIL_PATTERN)
 	GOOS=plan9 go test ./... | $(BUILD_FAIL_PATTERN)
+	CGO_ENABLED=0 GOOS=aix GOARCH=ppc64  go test ./... | $(BUILD_FAIL_PATTERN)
 
 ifeq ($(shell uname -s), Darwin)
 	CGO_ENABLED=1 GOOS=darwin go test ./... | $(BUILD_FAIL_PATTERN)
@@ -76,6 +77,8 @@ vet:
 
 	GOOS=plan9 GOARCH=amd64 go vet ./...
 	GOOS=plan9 GOARCH=386 go vet ./...
+
+	CGO_ENABLED=0 GOOS=aix GOARCH=ppc64 go vet ./...
 
 macos_test:
 	CGO_ENABLED=0 GOOS=darwin go test ./... | $(BUILD_FAIL_PATTERN)

--- a/host/host_aix.go
+++ b/host/host_aix.go
@@ -110,7 +110,7 @@ func parseUptime(uptime string) uint64 {
 		if err != nil {
 			return 0
 		}
-	case ut[3] == "mins," || ut[3] == "mins,":
+	case ut[3] == "mins,":
 		mins, err = strconv.ParseUint(ut[2], 10, 64)
 		if err != nil {
 			return 0

--- a/net/net_aix_cgo.go
+++ b/net/net_aix_cgo.go
@@ -30,7 +30,7 @@ func IOCountersWithContext(ctx context.Context, pernic bool) ([]IOCountersStat, 
 		iocounters = append(iocounters, n)
 	}
 	if pernic == false {
-		return getIOCountersAll(iocounters)
+		return getIOCountersAll(iocounters), nil
 	}
 	return iocounters, nil
 }

--- a/net/net_aix_nocgo.go
+++ b/net/net_aix_nocgo.go
@@ -89,7 +89,7 @@ func IOCountersWithContext(ctx context.Context, pernic bool) ([]IOCountersStat, 
 		return nil, err
 	}
 	if pernic == false {
-		return getIOCountersAll(iocounters)
+		return getIOCountersAll(iocounters), nil
 	}
 	return iocounters, nil
 }

